tests/gsea_test.py

```diff
@@ -0,0 +1,27 @@
+"""Verifies the 'gsea.py' module"""
+
+from nose.tools import ok_, raises
+from termlink.gsea import validate_path
+from termlink.gsea import generate_row
+
+@raises(ValueError)
+def test_validate_path_fails_on_extension():
```

Comment:
`yarn lint` is going to complain that there isn't method documentation for each of these. Mind adding it?

I haven't set up a minimum `pylint` threshold for passing, but it's on my todo list.